content/assignments/minishell/checker/README.md

```diff
@@ -0,0 +1,31 @@
+# Testing and Grading
+
+The testing is automated.
+Tests are located in the `inputs/` directory.
+
+```console
+student@so:~/.../assignments/minishell/checker/_test/inputs$ ls -F
```

Comment:
```suggestion
student@os:~/.../assignments/minishell/checker/_test/inputs$ ls -F
```

Reply:
Done since README is no longer in this folder.
It's already in the PR with the assignment.